updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,135 @@
 [id="updating-disconnected-cluster"]
-= Updating a disconnected cluster to a minor version
+= Updating a disconnected cluster
```

Comment:
"disconnected" -> "restricted network", here and throughout the content you're touching?  Previous thread [here][1].  And I realize that these `disconnected` references exist in master, but we're uncommenting them here, so I think we should move to the standard wording while we uncomment.

[1]: https://github.com/openshift/openshift-docs/pull/21993#discussion_r423267306